src/components/field/Field.vue

```diff
@@ -218,18 +217,11 @@ export default {
         * Set internal message when prop change.
         */
         message(value) {
-            this.newMessage = value
-        },
-
-        /**
-        * Set parent message if we use Field in Field.
-        */
-        newMessage(value) {
             if (this.parent && this.parent.hasInnerField) {
                 if (!this.parent.type) {
                     this.parent.newType = this.newType
                 }
-                this.parent.newMessage = value
+                this.parent.message = value
```

Comment:
Are you sure? You can't mutate a prop

Reply:
I ran tests for `Field.vue` and all passes. Also, I tested `BField` element nesting and it worked. Is there something wrong with my tests?

Reply:
Really strange that Vue doesn't show you warnings or similar